ext/soap/tests/bugs/gh16237.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+GH-16237 (Segmentation fault when cloning SoapServer)
+--EXTENSIONS--
+soap
+--FILE--
+<?php
+
+$server = new SoapServer(null, ['uri'=>"http://testuri.org"]);
```

Comment:
If we need to connect to a server, the test should be marked as offline test. Otherwise, me may use an `.invalid` domain here.

Reply:
This doesn't connect to a server.

Reply:
Then I suggest to use an invalid domain per [RFC 6761, sec. 6.4](https://datatracker.ietf.org/doc/html/rfc6761#section-6.4).